schema/job.json

```diff
@@ -16,4 +16,4 @@
     "required": [
         "workflow"
     ]
-}
+}
```

Comment:
https://stackoverflow.com/questions/44704968/visual-studio-code-insert-newline-at-the-end-of-files pls